### ISSUE TITLE
Remove Experimental from XPath Filter

### ISSUE THREAD

**Issue by rnveach** [checkstyle/checkstyle#11597]
XPath filters have been around since 8.3 in late 2017. We should remove the "experimental" part from them as they have been a normal part for a while.

https://github.com/checkstyle/checkstyle/search?q=experimental

**Comment by romani** [comment]
Yes, but now we have to support both names, as we lost a moment to get out of experimental.

But all in repo should be updated to not mention "experimental"

**Comment by Shaman20** [comment]
Is this issue still valid and can I work on it?

**Comment by rnveach** [comment]
Yes to both.

**Comment by Shaman20** [comment]
I am on it

**Comment by Shaman20** [comment]
It is throwing this error is there any other file which I have to edit?

![image](https://user-images.githubusercontent.com/70961605/168440936-b2bad6d4-6e45-48a2-be25-7c337cc39c7a.png)

**Comment by romani** [comment]
While you are blocked, please take some simple issue meanwhile

**Comment by romani** [comment]
we added new dtd files.
old dtd stays as is, and continue to be working might be forever, or some long time to keep compatibility.
